docs/runtimes/configuring-job-resources.md

```diff
@@ -365,7 +365,23 @@ The zone label from the project level is completely removed, and the resulting c
 ```
 {"gpu": "true", "arch": "amd64"}
 ```
+### Preventing conflicts
 
+If your function run is stuck with the status `pending`, it's possible that the "specified" node selector does not exist. There are three 
+levels of node selectors in MLRun: function, project, and service. At runtime, the system combines these selectors and applies the resolved 
```

Comment:
we have 4 levels of node selector: run, function, project, service

Reply:
I think this can be a little confusing for the users, since in most cases the run object is created at runtime from the function, and it's not something they are editing. The run level is only relevant for tasks and remote workflows and for the workflow runner, so If we are adding this here as part of the documentation we need to make sure it's clear enough

Reply:
@yaelgen how does the "run" node selector fit into the hierarchy of the other 3?

Reply:
let's keep it as is for now